zephyr/drivers/can/can_sccan.c

```diff
@@ -208,6 +231,15 @@ static int sc_can_enable(const struct sc_can_cfg *config)
 	return 0;
 }
 
+static inline bool sc_can_filter_is_used(const struct sc_can_cfg *config, int filter_id)
+{
+	if ((sc_can_read32(config, SCCAN_AFER_OFFSET) & BIT(filter_id))) {
```

Comment:
`sys_test_bit()`?

Reply:
Thanks. I didn't know `sys_test_bit()`, it is useful.
Will replace it.